### DISCUSSION TITLE
Formalized/automated changelog, versioning, release process?

### DISCUSSION THREAD

**Discussion by jstcki** [visualize-admin/visualization-tool#176] [Ideas]
Now that we're merging multiple PRs on a daily basis, should we formalize the process around how to describe changes, so a changelog can be generated automatically?

Cube Creator uses https://github.com/atlassian/changesets as far as I know. I haven't used it but seen it before and thought it would maybe be worth to give it a look?

**Comment by ptbrowne** [comment]
I already used a similar package for Python (towncrier), I like the fact that not every PR/commit gets to be inside the changelog. Looks promising !
